guides/common/modules/proc_importing-a-subscription-manifest-into-foreman-server.adoc

```diff
@@ -25,8 +25,15 @@ endif::[]
 endif::[]
 ifeval::["{mode}" == "disconnected"]
 * Ensure you have a Red{nbsp}Hat subscription manifest exported from the Red Hat Customer Portal.
+Use the same manifest that you use on the upstream {ProjectServer}.
 ifndef::orcharhino[]
-For more information, see {RHDocsBaseURL}subscription_central/1-latest/html/getting_started_with_rhel_system_registration/adv-reg-rhel-using-rhsm_#using_manifests_con[Using manifests for a disconnected Satellite Server] in _Subscription Central_.
+For more information, see {RHDocsBaseURL}subscription_central/1-latest/html-single/creating_and_managing_manifests_for_a_disconnected_satellite_server/index[Creating and managing manifests for a disconnected {ProjectServer}] in _Subscription Central_ and
+ifndef::content-management[]
+xref:configuring-server-to-synchronize-content-over-a-network_{context}[].
+endif::[]
+ifdef::content-management[]
+{InstallingServerDisconnectedDocURL}configuring-server-to-synchronize-content-over-a-network_satellite[Configuring {ProjectServer} to synchronize content over a network].
+endif::[]
```

Comment:
The `content-management` attribute will never be defined in this context because this part only appears in the _Installing Disconnected_ guide. So you can just go with:
```suggestion
xref:configuring-server-to-synchronize-content-over-a-network_{context}[].
```

Reply:
@Lennonka It can be (that's why my initial builds failed), here's the assembly containing this file:
`./guides/common/assembly_managing-red-hat-subscriptions.adoc\03:include::modules/proc_importing-a-subscription-manifest-into-foreman-server.adoc[leveloffset=+1]`

And this assembly is used inside Managing Content:
`./guides/doc-Managing_Content/master.adoc\029:include::common/assembly_managing-red-hat-subscriptions.adoc[leveloffset=+1]`

Reply:
[That](https://github.com/theforeman/foreman-documentation/pull/3505/commits/d1f798499037be34410c67d68785fb469c90ef16#diff-f12ced5de0b2f7e048059998927ae221a2e9d78eb9b458ad969cdf02511233ecR40) was the problem for the build, not this.